<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableClassClass</class>
 <widget class="QWidget" name="tableClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tableClass</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/7b78ceea29ae3cd6dbf27aa34bac4892.jpeg</normaloff>icon/7b78ceea29ae3cd6dbf27aa34bac4892.jpeg</iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1031</width>
     <height>781</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/bd2ca828946a79181a7b04b264820cc2(1).jpg</normaloff>icon/bd2ca828946a79181a7b04b264820cc2(1).jpg</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
